spec/jobs/stream_api_requests_to_big_query_job_spec.rb

```diff
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe StreamAPIRequestsToBigQueryJob, type: :job do
```

Comment:
I think the tests are a bit light here; we're missing coverage from lines 48, 57, 68 and 71 in the job that might be worth covering